docs/.vuepress/config.ts

```diff
@@ -19,6 +19,7 @@ export default defineUserConfig({
   alias: {
     // Alias src directory for @/components/{KongponentName} imports
     '@': path.resolve(__dirname, '../../src'),
+    utils: path.resolve(__dirname, '../../utils'),
```

Comment:
@adamdehaven willing ready and able to change this root folder name to something more appropriate :)